external/anomaly/tasks/inference.py

```diff
@@ -283,8 +283,11 @@ def save_model(self, output_model: ModelEntity) -> None:
         output_model.set_data("label_schema.json", label_schema_to_bytes(self.task_environment.label_schema))
         self._set_metadata(output_model)
 
-        f1_score = self.model.image_metrics.F1Score.compute().item()
-        output_model.performance = Performance(score=ScoreMetric(name="F1 Score", value=f1_score))
+        if hasattr(self.model, "image_metrics"):
+            f1_score = self.model.image_metrics.F1Score.compute().item()
+            output_model.performance = Performance(score=ScoreMetric(name="F1 Score", value=f1_score))
```

Comment:
I'm not sure exactly what this `output_model.performance` is used for, but shouldn't we report pixel-level performance when the task type is segmentation/detection?